web/elm/src/Pipeline/Styles.elm

```diff
@@ -23,16 +22,9 @@ groupsBar : List (Html.Attribute msg)
 groupsBar =
     [ style "background-color" Colors.groupsBarBackground
```

Comment:
Erm... why is this a block of styles and not a List of string string pairs? o.O

Reply:
[elm 0.19](https://discourse.elm-lang.org/t/style-from-v0-18-to-v0-19/1766)